pandas/core/frame.py

```diff
@@ -10563,13 +10563,13 @@ def isin(self, values) -> DataFrame:
 
         When ``values`` is a Series or DataFrame the index and column must
         match. Note that 'falcon' does not match based on the number of legs
-        in df2.
+        in other.
 
-        >>> other = pd.DataFrame({'num_legs': [8, 2], 'num_wings': [0, 2]},
+        >>> other = pd.DataFrame({'num_legs': [8, 3], 'num_wings': [0, 2]},
```

Comment:
Why did you change this?

Reply:
I didn't change these lines actually I only added the "." at the end of description of "check_dtype" and "obj". These changes are from a different issue that I solved and my PR got merged but I have no idea why these commits got pushed with this PR. You can have a look at this PR for better understanding - https://github.com/pandas-dev/pandas/pull/43820/files

Reply:
Do let me know what would I have to do in order to get rid of these extra commits.

Reply:
Simply revert the relevant commit on your branch

Reply:
something like
```
git fetch upstream
git rebase upstream/master
git push origin HEAD -f
```

Reply:
Done! Kindly review it again.